helpers/http_api.js

```diff
@@ -140,14 +140,20 @@ var middleware = {
 		if (!config.api.enabled) {
 			res
 				.status(apiCodes.INTERNAL_SERVER_ERROR)
-				.send({ success: false, error: 'API access disabled' });
+				.send({
+					message: 'API access disabled',
```

Comment:
@MaciejBaj @nazarhussain @4miners I feel this is a breaking change in API response, as the end user until now was expecting `status` and `error` message, now we change `error` -> `message` and new field `errors`?
whats your opinion?

Reply:
Our [API specification](https://github.com/LiskHQ/lisk/blob/development/schema/swagger.yml#L2434) says that the response for `AccessDeniedError` should be an object and must have `message` property. That being said, I reckon we are fixing a bug and not changing the expected API behaviour here.